### ISSUE TITLE
Visible players? Also mode documentation

### ISSUE THREAD

**Issue by DennHenda** [mcclure/emu-coop#7]
Is there any possibility players being able to see each other, even with no sort of actual interaction? Perhaps as simple as a sprite or harmless enemy that copies the partner's coordinates. I understand this may spam the irc even if it were possible, so it may not be a great idea anyway.

Also I'm working on a mode for MegaMan X, I'm sure I can get the addresses for bosses/weapons/upgrades, but I'm unsure of how to set up the rest of the Lua. I'm still only learning so I may figure it out in time, but a documentation would surely help!

Thanks for your work on this btw, it's an idea I had always dreamed of but never imagined possible.

**Comment by miketrethewey** [comment]
In response to Mega Man X, I found [a page](http://tasvideos.org/GameResources/SNES/MegaManX/RAMMap.html) from TASVideos that has some RAM information.

**Comment by DennHenda** [comment]
Yeah I've scrolled through that site quite a bit haha. I've also found www.retroachievements.org to be very helpful. Their emulators are customized with an "achievement editor" that lets you view the addresses used in their system.
I'm still working on MegaMan X, had a bit of help from Alvy to understand the structure of the Lua. I will probably do a pull request once I'm confident it works

**Comment by mcclure** [comment]
Mode documentation would of course be great D: i'll get on that as soon as i can.

Making other players visible would be basically impossible unless you modified the ROM. However, you *could* modify the ROM.

I'm talking to the zelda 3 randomizer crowd about adding some mechanisms that would allow someone with the ability to modify the game ROM to hook into the emu-coop communication channel.

**Comment by mcclure** [comment]
Custom messages (in 1.2) could help with this. There is also modefile documentation now. It's distributed with emu-coop, or see the website.
